src/modules/Dashboard/features/Applet/Popups/DuplicatePopups/DuplicatePopups.tsx

```diff
@@ -4,16 +4,30 @@ import { useForm } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 import * as yup from 'yup';
 import { yupResolver } from '@hookform/resolvers/yup';
+import { Box } from '@mui/material';
 
-import { Encryption, getEncryptionToServer, Mixpanel } from 'shared/utils';
+import {
+  Encryption,
+  getEncryptionToServer,
+  getPrivateKey,
+  Mixpanel,
+  publicEncrypt,
+} from 'shared/utils';
 import { Modal, Spinner, SpinnerUiType } from 'shared/components';
-import { InputController } from 'shared/components/FormComponents';
-import { StyledErrorText, StyledModalWrapper, variables } from 'shared/styles';
+import { CheckboxController, InputController } from 'shared/components/FormComponents';
+import { StyledErrorText, StyledFlexColumn, StyledModalWrapper, variables } from 'shared/styles';
```

Comment:
Needs `StyledBodyLarge` for use below
```suggestion
import {
  StyledBodyLarge,
  StyledErrorText,
  StyledFlexColumn,
  StyledModalWrapper,
  variables,
} from 'shared/styles';
```